google/cloud/bigquery/model.py

```diff
@@ -55,7 +55,7 @@ class Model(object):
     def __init__(self, model_ref):
         # Use _proto on read-only properties to use it's built-in type
         # conversion.
-        self._proto = types.Model()
+        self._proto = types.Model()._pb
```

Comment:
Could you explain why we're using `_pb`?

Reply:
`json_format.ParseDict()` used in methods such as `from_api_repr()` needs a raw protobuf message to work. We get the raw message from a proto-plus wrapper class through the `_pb` attribute.

Unifying the internal representations to raw protobuf messages was needed to fix several then-failing unit tests, e.g. for `Rotuine` and `RoutineReference`.

It might be achievable to store the actual proto-plus wrapper instances, but we would then have to place conversions to the raw protobuf messages at quite a few places around the code (but I didn't go down that route, had to pick one).

Reply:
Gotcha. There are a couple benefits to proto-plus (mostly around timestamp fields), but I think this is the pragmatic choice.